molgenis-questionnaires/src/main/java/org/molgenis/questionnaires/controller/QuestionnaireController.java

```diff
@@ -0,0 +1,64 @@
+package org.molgenis.questionnaires.controller;
+
+import org.molgenis.core.ui.controller.VuePluginController;
+import org.molgenis.core.ui.menu.MenuReaderService;
+import org.molgenis.questionnaires.response.QuestionnaireResponse;
+import org.molgenis.questionnaires.service.QuestionnaireService;
+import org.molgenis.security.user.UserAccountService;
+import org.molgenis.settings.AppSettings;
+import org.molgenis.web.PluginController;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+import static org.springframework.http.HttpStatus.OK;
+
+@Controller
+@RequestMapping(QuestionnaireController.URI)
+public class QuestionnaireController extends VuePluginController
```

Comment:
Add javadoc and or swagger docs to controller end points, these serve as a 'public' ( for app builder) api and should be documented.

Reply:
done